doc/interface/unit_operations/population_balance_model.rst

```diff
@@ -0,0 +1,159 @@
+.. _pbm_config:
+
+Population balance model
+=====================================
+
+One-dimensional (1D) PBM
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The 1D PBM is implemented in a CSTR as reactions. To configure the CSTR, see :ref:`cstr_config`.
```

Comment:
also here, the PBM is independent of the unit operation. please clarify.

Reply:
Another question about git: how do I clean up the commit history?